contracts/ColonyNetworkStorage.sol

```diff
@@ -59,20 +59,10 @@ contract ColonyNetworkStorage is DSAuth {
   // skillId of the root global skills tree
   uint256 rootGlobalSkillId;
 
-  struct ReputationLogEntry {
-    address user;
-    int amount;
-    uint256 skillId;
-    address colony;
-    uint256 nUpdates;
-    uint256 nPreviousUpdates;
-  }
-
-  mapping (uint => ReputationLogEntry[]) reputationUpdateLogs;
-  // Tracks whether updateLog 0 or 1 is the active log
-  uint256 activeReputationUpdateLog;
   // Address of the currently active reputation mining cycle contract
-  address reputationMiningCycle;
+  address currentReputationMiningCycle;
+  // Address of the next active reputation mining cycle contract, which is where new reputation updates are put.
+  address nextReputationMiningCycle;
```

Comment:
These names confused me. The "current" and "next" definitions differ depending on who you are, for a rep miner, the above makes sense, but for the network, the current log is the one which it appends updates to. I am struggling to come up with some alternatives but maybe `minerReputationMiningCycle` and `networkReputationMiningCycle`?

Reply:
I went with the perspective of the reputation miner, as the contract in question is `ReputationMiningCycle`, which is definitely more concerned with the state of reputation mining than which logs are currently being appended to.

I agree this isn't perfect (which is why I called it out in the PR description), but I wasn't able to come up with anything better. I prefer this nomenclature to `minerReputationMiningCycle` and `networkReputationMiningCycle`, though am open to further suggestions.

Reply:
Some more ideas then, in order of preference:
`activeReputationMiningCycle` / `inactiveReputationMiningCycle` or `passiveReputationMiningCycle`
`reputationMiningCycle` / `nextReputationMiningCycle`
`runningReputationMiningCycle` / `dormantReputationMiningCycle`

Also would prefer a single getter like before.

Reply:
I am happy with `active`/`inactive`, I think, so I'll switch to that.